libsolidity/codegen/ABIFunctions.cpp

```diff
@@ -1566,3 +1559,28 @@ size_t ABIFunctions::headSize(TypePointers const& _targetTypes)
 	return headSize;
 }
 
+string ABIFunctions::suffixedVariableNameList(string const& _baseName, size_t _startSuffix, size_t _endSuffix)
+{
+	string result;
+	if (_startSuffix < _endSuffix)
+	{
+		result = _baseName + to_string(_startSuffix++);
+		while (_startSuffix < _endSuffix)
+			result += ", " + _baseName + to_string(_startSuffix++);
+	}
+	else if (_endSuffix < _startSuffix)
+	{
+		result = _baseName + to_string(_endSuffix++);
+		while (_endSuffix < _startSuffix)
+			result = _baseName + to_string(_endSuffix++) + ", " + result;
+	}
+	return result;
+}
+
+size_t ABIFunctions::numVariablesForType(Type const& _type, EncodingOptions const& _options)
+{
+	if (!_options.encodeFunctionFromStack && _type.category() == Type::Category::Function)
+		return 1;
```

Comment:
Reading the comment from the header file and this I'm confused if it should be 1 or 2.